### DISCUSSION TITLE
ASIO support

### DISCUSSION THREAD

**Discussion by Gr33nEyes** [audacity/audacity#1809] [Q&A] [answered]
I want to build Audacity with ASIO support
How can I do this?

**Comment by LWinterberg** [comment, marked as answer]
https://manual.audacityteam.org/man/asio_audio_interface.html has some (somewhat outdated) info on it. Maybe that's enough to get you started.

**Comment by werame** [comment]
The wiki info is pretty outdated. The current conan distro of portaudio will auto-download the ASIO SDK if built (via Audacity) with the `-Daudacity_has_asio_support=On`. 

https://github.com/audacity/audacity/blob/master/BUILDING.md#building-with-asio-support-on-windows actually comes closest to describing how it should be done nowadays.
